### DISCUSSION TITLE
3.3.1 Release

### DISCUSSION THREAD

**Discussion by snowman2** [pyproj4/pyproj#1063] [General]
https://github.com/pyproj4/pyproj/releases/tag/3.3.1rc0

The plan is to release with PROJ 8.2 in the wheels.

Release Candidate 0 Status:

- [x] Linux/OSX wheels (https://github.com/pyproj4/pyproj/actions/runs/2199332740; https://github.com/pyproj4/pyproj-wheels/pull/53)
- [x] Windows wheels (@cgohlke)
- [x] Conda Forge (https://github.com/conda-forge/pyproj-feedstock/pull/119)
- [x] Debian (@sebastic)
- [x] Fedora (@jdkloe)

Testing is always appreciated, but definitely not required.

**Comment by sebastic** [comment]
The Debian package built fine:

 https://buildd.debian.org/status/package.php?p=python-pyproj&suite=experimental

**Reply by snowman2** [reply]
Thanks @sebastic 👍

**Comment by snowman2** [comment]
3.3.1 is released and on pypi

**Reply by jdkloe** [reply]
Hi @snowman2, I am missing the tar.gz source package on pypi. Could you publish that one as well?

**Reply by snowman2** [reply]
Thanks - it should be there now.

**Reply by jdkloe** [reply]
Thanks for providing the tar file (my fedora packaging workflow depends on it).
Also I'm sorry I had no opportunity to test 3.3.1 before you released it.
I have built and installed the new version now in fedora 35, 36 and 37 and this works just fine.

**Reply by snowman2** [reply]
Thank you 👍 - I will make sure for the 3.4 release that there is a bigger gap between the release candidate and the release.
